build/vaultwarden/local.mog

```diff
@@ -0,0 +1,26 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+
+# Copyright 2022 OmniOS Community Edition (OmniOSce) Association.
+
+dir group=$(GROUP) mode=0770 owner=$(USER) path=var/$(BASEDIR)
+
+group groupname=$(GROUP) gid=58
+user ftpuser=false username=$(USER) uid=58 group=$(GROUP) \
+    gcos-field="vaultwarden - Password Manager" \
+    home-dir=/var/$(BASEDIR) passwd=NP
+
+# XXX what goes here?
+#<transform file path=etc/ -> set preserve renamenew>
```

Comment:
Not quite sure what to put here yet as vaultwarden only cares about the files under /var/ooce

Reply:
I'd move the config file to `/etc` and symlink it from `/var`